pkg/universe.dagger.io/go/image.cue

```diff
@@ -20,6 +20,16 @@ _#DefaultVersion: "1.18"
 			docker.#Pull & {
 				source: "index.docker.io/golang:\(version)-alpine"
 			},
+			docker.#Run & {
+				command: {
+					name: "apk"
+					args: ["add", "git"]
+					flags: {
+						"-U":         true
+						"--no-cache": true
+					}
+				}
+			},
```

Comment:
Instead of adding whole `docker.#Run` operation, you could simply add `packages: "git": _` below `packages` declaration.

And it will be installed by the for loop below.

Reply:
But I agree with Helder, that would be better to rely on official go image and add any package that you need in a custom `go.#Image`

Reply:
I tend to agree with both of you (also, I learned the `: _` approach today yay).